integration_tests/test_data.py

```diff
@@ -1,163 +1,173 @@
-import os
```

Comment:
I don't understand what happened here... the diff is nuts, I can't see what is new and what isn't..

Reply:
ugh, yes.  I had to reinstall git and might have introduced a line ending mismatch or something.  I can close and roll back commits to try and triage.